packages/cloud-config/oem/10_selinux.yaml

```diff
@@ -0,0 +1,16 @@
+# Default cOS OEM configuration file
+# Default cOS OEM configuration file
+#
+# This file is part of cOS and will get reset during upgrades.
+#
+# Before you change this file manually,
+# consider copying this file to /usr/local/cloud-config or
+# copy the file with a prefix starting by 90, e.g. /oem/91_custom.yaml
+name: "SELinux"
+stages:
+   initramfs:
+     - name: "Relabelling"
+       commands:
+       - |
+           load_policy -i
+           restorecon -R -i -v /etc /root /opt /srv /var /home /usr/local /oem
```

Comment:
This doesnt need an input file? How does that work? uses the default file in the selinux dir (/var/selinux or whatever)